tests/unit/amo/reducers/test_collections.js

```diff
@@ -444,6 +451,24 @@ describe(__filename, () => {
       expect(collectionInState.addons)
         .toEqual(createInternalAddons(addons));
     });
+
+    it('loads notes for collection add-ons', () => {
+      const notes = 'These are some notes.';
+      const fakeCollectionAddon =
+        createFakeCollectionAddon({ notes });
+      const addons = createFakeCollectionAddons({
+        addons: [fakeCollectionAddon],
+      });
+      const collection = createFakeCollectionDetail({ addons });
+
+      const state = loadCollectionIntoState({
+        state: initialState, collection, addons,
+      });
+
+      const collectionInState = state.byId[collection.id];
+      expect(collectionInState.addons[0].collectionNotes)
+        .toEqual(notes);
```

Comment:
In general, it is better to use the full available width. 80 chars is not a lot, but these two lines should have less chars than that 🤓